features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/CreateScope.jsx

```diff
@@ -185,9 +185,18 @@ class CreateScope extends React.Component {
 
     validateScopeDescription({ target: { id, value } }) {
         const { valid, apiScope } = this.state;
+        const { intl } = this.props;
         apiScope[id] = value;
-        valid[id].invalid = false;
-        valid[id].error = '';
+        if (value && value.length !== '' && value.length >= 512) {
```

Comment:
It seems like the `value` variable is holding an object here, If so `value.lenngth` would be undefined and will never execute the if condition?

Reply:
value.length is coming correctly
<img width="513" alt="Screen Shot 2020-08-18 at 4 12 34 PM" src="https://user-images.githubusercontent.com/19324135/90503844-c1f58780-e16d-11ea-9ff7-2fc88e1b5374.png">

Reply:
yes correct, i mixed up the two variables 😄  `value` vs `valid`